cmake/winml_unittests.cmake

```diff
@@ -55,6 +55,8 @@ endfunction()
 
 file(GLOB winml_test_common_src CONFIGURE_DEPENDS "${WINML_TEST_SRC_DIR}/common/*.cpp")
 add_library(winml_test_common STATIC ${winml_test_common_src})
+add_dependencies(winml_test_common onnx)
```

Comment:
What does the onnx target do?

Reply:
protobufHelpers.cpp includes onnx/onnx-ml.pb.h which wasn't resolved until depending on the onnx project.